### ISSUE TITLE
I don't quite understand the combine rows method

### ISSUE THREAD

**Issue by rebuilt** [edcommonwealth/merge-csv#2]
https://github.com/edcommonwealth/merge-csv/blob/73cfb0ad7765f94e852f69ffc7e88c63c9bd4390/merge-csv.py#L120

I'm having a hard time figuring out if your combine rows method will combine the 'RecordedDate' column?   Will this method cycle through all permutations of every possible letter in the text?

@mnrva-dev

**Comment by mnrva-dev** [comment]
The method takes a dataframe, a final column name, and a list of all possibilities that the column can have. It then iterates through the dataframe's columns checking for any of the listed possibilities, as well as any permutation of uppercase and lowercase letters of each possibility (so long as the possibility contains fewer than 15 characters - it would take too long otherwise). Then, when it finds a column that is listed as a possibility, it merges it under the final column label.
@rebuilt

**Comment by rebuilt** [comment]
Instead of checking permutations of each of the letters in the name, wouldn't it be more efficient to lowercase each of the possibilities and checking the lowercased version of the column name to see if they match?  e.g "ResponseId".lower() == "responseID".lower()

@mnrva-dev

**Comment by mnrva-dev** [comment]
Yes, it would be, and I'm embarrassed not to have thought of that in the first place. Doing it now.
